web/src/containers/ApdStateProfileMedicaidOffice.js

```diff
@@ -21,68 +22,83 @@ class ApdStateProfile extends Component {
 
     return (
       <Fragment>
-        <h3 className="mt0">{t(`${dirTRoot}.title`)}</h3>
-        <div className="mb3">
-          <Input
+        <h4 className="ds-h4 ds-u-margin-bottom--0">
```

Comment:
Not 100% sure on the need for this 0 margin bottom.  There isn't much margin anyway, but looking at Jen's screenshots, it looked like there should be less.

Reply:
It occurs to me now that we could just nix the `h4` and use the `fieldset legend` as the section title. Suggested code incoming!